src/main/java/Temp/MemberDBBean.java

```diff
@@ -1,5 +1,30 @@
 package Temp;
 
+import java.util.List;
+import java.util.Map;
+
+import org.apache.ibatis.session.SqlSession;
+
+import bean.SqlMapClient;
+import Temp.UserDataBean;
+
 public class MemberDBBean {
+	private SqlSession session=SqlMapClient.getSession();
 
+	public List<UserDataBean> getCurrentMember(String pao_trip_id) {
+		List<UserDataBean> memberList=session.selectList("user.getCurrentMember", pao_trip_id);
+		for(UserDataBean user:memberList) {
+			user.setUser_name((String)session.selectOne("user.getUserName", user.getUser_id()));
+		}
+		return memberList;
+	}
+	public int addTripMember(Map<String, String> addMemberMap) {
+		return session.update("user.addTripMember", addMemberMap);
+	}	
+	public int delTripMember(Map<String, String> delMemberMap) {
+		return session.update("user.delTripMember", delMemberMap);
+	}	
+	public int isMember(Map<String, String> delMemberMap) {
+		return session.selectOne("user.isMember2", delMemberMap);
```

Comment:
```suggestion
		return session.selectOne("user.isMember", delMemberMap);
```
xml에 있는게 isMember2 인데 정작 DBBean 에 있는 메서드 이름은 isMember 잖아요
이런 부분들 한가지로 통일해주세요
cf) 보실 때 핸들러에서 메서드 콜하는 것도 찾아보셔야 합니다